docs/resources/User.md

```diff
@@ -33,6 +33,8 @@ There are other rules and restrictions not shared here for the sake of spam and
 | bot?          | boolean   | whether the user belongs to an OAuth2 application                                                    | identify              |
 | system?       | boolean   | whether the user is an Official Discord System user (part of the urgent message system)              | identify              |
 | mfa_enabled?  | boolean   | whether the user has two factor enabled on their account                                             | identify              |
+| banner?       | ?string   | the user's banner, or null if unset                                                                  | identify              |
+| banner_color? | ?number   | the user's banner color as a hex string prefixed with "#", or null if unset                          | identify              |
```

Comment:
Shouldn't this be `?string`

Reply:
```suggestion
| banner_color? | ?string   | the user's banner color as a hex string prefixed with "#", or null if unset                          | identify              |
```

Reply:
Yes, I wrote that before I realized it's a hex string lol